config/params.php

```diff
@@ -2,15 +2,23 @@
 
 declare(strict_types=1);
 
+use App\Command\Hello;
 use App\ApplicationParameters;
+use Yiisoft\Arrays\Modifier\ReverseBlockMerge;
 use Yiisoft\Assets\AssetManager;
 use Yiisoft\Factory\Definitions\Reference;
-use Yiisoft\Form\Widget\Field;
+use Yiisoft\I18n\Locale;
 use Yiisoft\Router\UrlGeneratorInterface;
 use Yiisoft\Router\UrlMatcherInterface;
 use Yiisoft\Yii\View\CsrfViewInjection;
 
 return [
+    'app' => [
+        'charset' => 'UTF-8',
+        'language' => 'en-US',
```

Comment:
How about naming it `locale`? It's not a language since it contains `US`, a country and may contain more.

Reply:
Ok, i'm going to use the ISO code for the English language `en`.